addons/mail/static/src/backend_components/activity_list_view_item/activity_list_view_item.scss

```diff
@@ -0,0 +1,15 @@
+.o_ActivityListViewItem {
+    .o_ActivityListViewItem_actionButtonContainer a {
```

Comment:
if `a` needs to be a target, prefer adding classes on the links that need the style specifically